distribution/changelog.txt

```diff
@@ -5,6 +5,7 @@
 - Feature: [#12712] Add TCP / socket plugin APIs.
 - Feature: [#12840] Add Park.entranceFee to the plugin API.
 - Feature: [#12886] Make all scenery placement and remove actions available to the plugin API.
+- Feature: [#12922] Add snow, heavy snow and blizzard to weather types.
```

Comment:
add #2350 and #10754

Reply:
I think just mentioning #2350 should do.